[2022-03-18 13:48:28,446] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2022-03-18T08:16:05.696376+00:00 [queued]>
[2022-03-18 13:48:28,489] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2022-03-18T08:16:05.696376+00:00 [queued]>
[2022-03-18 13:48:28,490] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 13:48:28,490] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-03-18 13:48:28,494] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 13:48:28,564] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2022-03-18T08:16:05.696376+00:00
[2022-03-18 13:48:28,572] {standard_task_runner.py:52} INFO - Started process 12538 to run task
[2022-03-18 13:48:28,595] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2022-03-18T08:16:05.696376+00:00', '--job-id', '18', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpjl4yww8i', '--error-file', '/tmp/tmp__4efwp4']
[2022-03-18 13:48:28,598] {standard_task_runner.py:77} INFO - Job 18: Subtask task_4
[2022-03-18 13:48:28,739] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2022-03-18T08:16:05.696376+00:00 [running]> on host pc
[2022-03-18 13:48:28,869] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2022-03-18T08:16:05.696376+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-18T08:16:05.696376+00:00
[2022-03-18 13:48:28,870] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-03-18 13:48:28,871] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-03-18 13:48:28,882] {subprocess.py:75} INFO - Output:
[2022-03-18 13:48:31,885] {subprocess.py:83} INFO - Command exited with return code 0
[2022-03-18 13:48:31,962] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20220318T081605, start_date=20220318T081828, end_date=20220318T081831
[2022-03-18 13:48:32,029] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-18 13:48:32,053] {local_task_job.py:151} INFO - Task exited with return code 0
